cli/cli.go

```diff
@@ -1,456 +1,822 @@
-				return []error{err}
-			}
-			exitCode, err := func(exitCh <-chan dockerct.ContainerWaitOKBody, errCh <-chan error) (int64, error) {
-				for {
-					select {
-					case exitOk := <-exitCh:
-						return exitOk.StatusCode, nil
-					case err := <-errCh:
-						return -1, err
-					}
-				}
-			}(exitCh, errCh)
-			if err != nil {
-				return []error{err}
-			}
-			if exitCode != 0 {
-				return []error{fmt.Errorf("%s ends with unexpected status code %d", services.ServiceCtValidatorImport, exitCode)}
-			}
-			if err := sedgeActions.ImportSlashingInterchangeData(actions.SlashingImportOptions{
-				ValidatorClient: flags.validatorName,
-				Network:         flags.network,
-				StopValidator:   false,
-				StartValidator:  false,
-				GenerationPath:  flags.generationPath,
-				From:            flags.slashingProtection,
-			}); err != nil {
-				return []error{err}
-			}
-		}
-		if err = runAndShowContainers(cmdRunner, *flags.services, flags); err != nil {
-			return []error{err}
-		}
-	} else {
-		// Let the user decide to see the instructions for executing the scripts and exit or let the tool execute them
-		if err = runScriptOrExit(cmdRunner, flags); err != nil {
-			return []error{err}
-		}
+func inputConsensusBootNodes(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	bootNodesInput, err := p.Input("Consensus boot nodes", "", false)
+	if err != nil {
+		return err
 	}
+	bootNodesList := strings.Split(bootNodesInput, ",")
+	o.genData.CCBootnodes = &bootNodesList
+	return nil
+}
 
+func inputMevImage(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// Default value is set in the template
+	o.genData.MevImage, err = p.Input("Mev-Boost image", "flashbots/mev-boost:latest", false)
+	return
+}
+
+func inputMevBoostEndpoint(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.genData.MevBoostEndpoint, err = p.Input("Mev-Boost endpoint", "", false)
+	return
+}
+
+func inputRelayURL(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// TODO add default relay URL value, it is not present in the generate command
+	o.genData.RelayURL, err = p.Input("Relay URL", "", false)
+	return
+}
+
+func inputGraffiti(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	o.genData.Graffiti, err = p.Input("Graffiti", "", false)
+	return
+}
+
+func inputCheckpointSyncURL(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// Default value is set in the template
+	o.genData.CheckpointSyncUrl, err = p.Input("Checkpoint sync URL", "", false)
+	return
+}
+
+func inputFeeRecipient(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	// TODO: Is necessary to add a default value?
+	o.genData.FeeRecipient, err = p.EthAddress("Please enter the Fee Recipient address. You can leave it blank and press enter (not recommended)", "")
+	return
+}
+
+func inputValidatorGracePeriod(p prompter.Prompter, o *CliCmdOptions) (err error) {
+	epochs, err := p.InputInt64("Validator grace period (epochs)", 1)
```

Comment:
Explain this value better (a new user should know what is this about). You can be comprehensive.

Reply:
Done https://github.com/NethermindEth/sedge/pull/228/commits/40cd8c8adeb6ed3675638aa6e593f354455605c6